Apps/Database/src/Delegates/DbResourceDelegateDelegate.cs

```diff
@@ -87,7 +87,7 @@ public DbResult<ResourceDelegate> Insert(ResourceDelegate resourceDelegate, bool
         }
 
         /// <inheritdoc/>
-        public DbResult<IEnumerable<ResourceDelegate>> Get(string delegateId, int page, int pageSize)
+        public DbResult<IEnumerable<ResourceDelegate>> Get(string delegateId, int page = 0, int pageSize = 500)
```

Comment:
Not worth rejecting but for thought:

What I've been hearing about the normal admin page size is only around 25 max. Should we be setting such a high default.

Reply:
I fixed this in another branch